src/Templates/parts/table-footer.twig

```diff
@@ -1,8 +1,8 @@
 <script type="text/javascript" src="{{baseurl}}js/vendor/jquery.dataTables.min.js"></script>
+<script type="text/javascript" src="{{baseurl}}js/vendor/dataTables.bootstrap.js"></script>
 <script type="text/javascript">
   //Set DataTable defaults
   $.extend($.fn.dataTable.defaults, {
-    bJQueryUI: true,
```

Comment:
It hasn't been. A long time ago it was in every file and removed. Now it's not using bJQueryUI - it's bootstrap now!